src/data/mainnet/celo-tokens-info.json

```diff
@@ -120,22 +120,22 @@
   {
     "address": "0x2a3684e9dc20b857375ea04235f2f7edbe818fa7",
     "decimals": 6,
-    "name": "USD Coin",
+    "name": "USD Coin (Truefeedback)",
     "symbol": "USDC"
   },
   {
     "address": "0x2def4285787d58a2f811af24755a8150622f4361",
     "decimals": 18,
     "imageUrl": "https://raw.githubusercontent.com/valora-inc/address-metadata/main/assets/tokens/cETH.png",
-    "name": "Wrapped Ethereum",
+    "name": "Wrapped Ethereum (Compound Finance)",
```

Comment:
Where do we go to actually bridge ETH and get this token?

Reply:
With this one I was more unsure... I added the name because in celo scan the token is labeled as `Compound Finance: cETH Token`.
It links to this page: https://compound.finance/, but I don't see Celo in the networks...